lib/pharos/config_schema.rb

```diff
@@ -89,6 +89,9 @@ def self.messages
           optional(:mode).filled(included_in?: %w(userspace iptables ipvs))
         end
         optional(:addons).value(type?: Hash)
+        optional(:kubelet).schema do
+          optional(:read_only_port).filled(:bool?)
```

Comment:
Should this be an integer (default `0`)?

Reply:
Right, maybe it's easier to just have a boolean for now.

Reply:
yeah, that's the reason I went with boolean for now :)